functions/package.json

```diff
@@ -26,7 +26,7 @@
     "jsonresume-theme-classy": "^1.0.9",
     "jsonresume-theme-cora": "^0.1.1",
     "jsonresume-theme-dave": "^1.0.1",
-    "jsonresume-theme-elegant": "^1.13.2",
+    "jsonresume-theme-elegant": "^1.15.1",
```

Comment:
Please update it to `1.16.1` :D 

I've fixed some v1.0 schema breaking changes